History.txt

```diff
@@ -7,6 +7,7 @@ Enhancements
 - accessor methods for `backtrace` settings [PR #134]
 - asynchronous writing from buffered appenders [PR #135]
 - improve date format performance when outputting microseconds [PR #136]
+- added some thread safety to log level setters [PR #137]
```

Comment:
Just a suggestion: have you looked into the [github_changelog_generator](https://github.com/skywinder/github-changelog-generator) project? It makes maintaining a history file like this a much nicer experience.